crates/evm/src/evm/db.rs

```diff
@@ -44,7 +44,8 @@ impl<'a, C: sov_modules_api::Context> Database for EvmDb<'a, C> {
         let bytecode = Bytecode::new_raw(
             self.code
                 .get(&code_hash, self.working_set)
```

Comment:
shouldn't this whole function now be 
```rust
return self.code.get(&code_hash, self.working_set).unwrap_or_default()
```

if evm db wants us to commit bytecode type 

we should keep it as is and return it as is 

because revm does some work on the bytecode and wants to keep that work